### ISSUE TITLE
Why the seed is 14159265?

### ISSUE THREAD

**Issue by fingertap** [hkchengrex/STCN#114]
This seed seems pretty weird to me. Why setting the seed to this specific number?

**Comment by hkchengrex** [comment]
Digits of pi after the decimal place.
I remember reading somewhere that it is better to set a large seed so I set it to that instead of 0.
It also follows from my previous work MiVOS: https://github.com/hkchengrex/Mask-Propagation/blob/ec9309f04ae3c98edb2eb5675c937a699d80006f/train.py#L26

**Comment by fingertap** [comment]
I see.

**Comment by hkchengrex** [comment]
The "large seed" claim is from here: https://pytorch.org/docs/1.10/generated/torch.Generator.html#torch.Generator.manual_seed
I don't know why though.

**Comment by fingertap** [comment]
FYI, https://discuss.pytorch.org/t/torch-generator-manual-seed-it-is-recommended-to-set-a-large-seed-why/120819
